app-commons/src/main/scala/org/bitcoins/commons/ConfigFactory.scala

```diff
@@ -0,0 +1,28 @@
+package org.bitcoins.commons
+
+import java.io.File
+import java.nio.file.Path
+
+trait ConfigFactory[T] {
```

Comment:
nit: I don't think there is any reason this can't be moved into `org.bitcoins.core.api.commons`